src/app/lib/config/services/loadableConfig.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This config is used by `createLoadableContext`
+ * to dynamically load each service's config.
+ */
+const loadableConfig = {
+  news: {
+    loader: () => import(/* webpackChunkName: 'news' */ './news.js'),
+    webpack: () => [require.resolveWeak('./news.js')],
```

Comment:
the react-loadable docs say to also for each service in this file add a `modules: ['./news.js'],` - my understanding is that we don't need it because that's only used in react-loadable's reporting of which bundles it used in the SSR, which we don't care about - I just want to check that's right?

Reply:
Yeah, were doing that all ourselves :)